less/home-page.less

```diff
@@ -1 +1,119 @@
-// Home Content
\ No newline at end of file
+// Home Content
+.home {
+  .flex;
+  flex-flow: column wrap;
+  margin: 0 auto;
+  max-width: 880px;
+  width: 100%;
+
+  @media @phone {
+    flex-flow: column nowrap;
+    width: 100%;
+  }
+
+  .top-section {
+    img {
+      width: 100%;
+    }
+  }
+
+  .text-section {
+    @media @phone {
+
+      width: 100%;
+    }
+   h2 {
+     padding: 1rem;
+     font-size: @main-h2-size;
+   }
+  }
+
+  .center-images {
+    .flex;
+    width: 100%;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+     @media @phone {
+       width: 100%;
+       align-self: center;
+       padding: 1rem;
+     }
+
+    img {
+      padding: .5rem;
+      width: 33%;
+      height: 16rem;
+      .border-radius(30px);
+
+         @media @phone {
+           width: 100%;
```

Comment:
You could also add `height: auto;` in the media query to keep the size constraint.